build.gradle

```diff
@@ -15,6 +15,12 @@ plugins {
 version="1.1.0"
 group = "gov.cms.ab2d"
 
+sonar {
+  properties {
+    property "sonar.projectKey", "ab2d-events"
```

Comment:
Instead of setting this can we update the rootProject.name?

https://github.com/CMSgov/ab2d-events/blob/79030c8740a34216f9f7dc3662ef023b3c471f2a/settings.gradle#L15

We may want to test a deploy of ab2d-events with this change in a lower environment and get @Sadibhatla to take a look in case there are side effects.